libraries/Bluefruit52Lib/src/BLEDiscovery.cpp

```diff
@@ -109,7 +109,8 @@ uint8_t BLEDiscovery::discoverCharacteristic(uint16_t conn_handle, BLEClientChar
   // We could found more characteristic than we looking for. Buffer must be large enough
   enum { MAX_DISC_CHARS = 4 };
```

Comment:
maybe we should increase this to 8, discovery is expensive operation and takes air time. I wasn't too sure which is max each pass. Look like the sizeof(ble_gattc_char_t) is not that big.